src/Umbraco.Cms.Api.Management/DependencyInjection/MemberBuilderExtensions.cs

```diff
@@ -10,7 +10,7 @@ internal static class MemberBuilderExtensions
 {
     internal static IUmbracoBuilder AddMember(this IUmbracoBuilder builder)
     {
-        builder.Services.AddTransient<IMemberPresentationFactory, MemberPresentationFactory>();
+        builder.Services.AddSingleton<IMemberPresentationFactory, MemberPresentationFactory>();
```

Comment:
I don't think it qualifies as a breaking change. And yeah, anything that is stateless should be singleton IMO, unless there is a really good reason otherwise. But that's a whole other PR 😆